src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -291,35 +294,35 @@ def _delete_blob(blob_name):
     return [_delete_blob(blob) for blob in source_blobs]
 
 
-def _copy_blob_to_blob_container(blob_service, source_blob_service, destination_container,
+def _copy_blob_to_blob_container(blob_service, source_blob_service, destination_container, destination_path,
                                  source_container, source_sas, source_blob_name):
     from azure.common import AzureException
     source_blob_url = source_blob_service.make_blob_url(source_container, encode_for_url(source_blob_name),
                                                         sas_token=source_sas)
-
+    destination_blob_name = create_normalized_blob_file_path(destination_path, source_blob_name)
```

Comment:
`create_normalized_blob_file_path` => `normalize_blob_name`

Reply:
This is used for both blob and file paths, not just for blobs.
For blobs, this "path" is part of the name, hence my lengthier naming to avoid confusion.

Reply:
ok so how about `normalize_blob_file_path`? The point here is __create_normalized__ equals __normalize__.